.github/workflows/loki.yaml

```diff
@@ -0,0 +1,70 @@
+name: Push to Loki
+
+on:
+  workflow_run:
+    workflows:
+      - BPF checks
```

Comment:
just curious if these are all workflows we have, if yes, I am wondering if there is anything like * syntax.

Reply:
yeah these are all the workflows from master, v1.12, v1.11, and v1.10 branches. no wildcard unfortunately https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run

Reply:
thanks man :100: